### PR TITLE
[WIP] Peer review for chapters 5 and 6

### PR DESCRIPTION
**Pull request by mishaone**
Updated with Petra's feedback for chapters 5 and 6. Waiting for conclusion of feedback.

### CODE REVIEW

**Comment by rkratky** [comment]
@mishaone, is this good to be merged? If not, could you please add `[WIP]` to the title?

**Comment by mishaone** [comment]
Can't tag her here, so asking Petra to review the changes here and give the OK to merge.

**Comment by mishaone** [comment]
*phew* I got a weird merge conflict, which I have now fixed but I don't want to lose the changes here or cause any more conflicts, so I'll just merge this. Petra, if you have any other changes, please let me know in the JIRA and I'll just create a new branch.

**Comment by petra-sargent** [comment]
I made it over here to review but I am going to have to get a refresher on GitHub!!!

**Comment by mishaone** [comment]
@petra-sargent, if you click the *Files Changed* tab up top, you can see a diff by file. Hopefully that's enough to get an idea of the changes.

File: docs/topics/modules/assigning_an_area.adoc [hunk 1]
```diff
@@ -10,6 +10,6 @@ You can group Work Items by type or functionality by assigning relevant Areas to
 .Procedure
 . Click the btn:[Plan] tab to view a list of Work Items for your Space.
 . Double-click a Work Item to view its details.
-. Click the *Area* text box to view a list of Areas to assign to the space. By default, your Root Area is assigned to a new Work Item.
+. Click the *Area* field to view a list of Areas to assign to the space. By default, your Root Area is assigned to a new Work Item.
```
Comment (petra-sargent) [inline docs/topics/modules/assigning_an_area.adoc]: I just tested the instructions and it look good!

File: docs/topics/modules/associating_work_items_with_an_iteration.adoc [hunk 2]
```diff
@@ -3,12 +3,14 @@
 
 To associate a Work Item with an Iteration:
 
-. Click the three dots adjoining the Work Item and click btn:[Associate with Iteration] from the displayed options. The `Associate with Iteration` dialog box appears and displays which Iteration, if any, the Work Item is associated with.
+. Click btn:[Plan] to see the Planner for your Space.
```
Comment (petra-sargent) [inline docs/topics/modules/associating_work_items_with_an_iteration.adoc]: I'm having a hard time finding the Plan button..I am on the Plan tab and I added a work item. Where should I see the Plan button. Thanks!

File: docs/topics/modules/creating_a_new_space.adoc [hunk 3]
```diff
@@ -11,9 +11,9 @@ To start working with a new project, create a new Collaboration Space. Each Spac
 
 . On your {osio-link} home page, click your user name in the top left corner of the screen.
```
Comment (petra-sargent) [inline docs/topics/modules/creating_a_new_space.adoc]: Theses changes look good !! I tested them!! :-)

File: docs/topics/modules/modifying_an_existing_work_item.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [#modifying_an_existing_work_item]
 = Modifying an Existing Work Item
 
-Modify an existing Work Item by double-clicking on it in the `Backlog` or `Board` view. Alternatively, click the icon adjoining the Work Item and select *Open* from the displayed options. The detailed Work Item panel is displayed.
+Modify an existing Work Item by double-clicking on it in the `Backlog` or `Board` view. Alternatively, click the icon (image:kabob_white.png[title="Options"]) adjoining the Work Item and select *Open* from the displayed options. The detailed Work Item panel is displayed.
 
 You can modify the following Work Item attributes:
```
Comment (petra-sargent) [inline docs/topics/modules/modifying_an_existing_work_item.adoc]: Looks good!

File: docs/topics/modules/quick_add_work_item.adoc [hunk 5]
```diff
@@ -3,14 +3,14 @@
 
 You can use the quick-add method as an instant widget to add Work Items to your Collaboration Space:
 
-. Click btn:[Plan] to view the *Planner* for your Space.
+. Click btn:[Plan] to view the `Planner` for your Space.
```
Comment (petra-sargent) [inline docs/topics/modules/quick_add_work_item.adoc]: Having trouble finding the Plan button..

File: docs/topics/modules/using_backlog_to_create_work_item.adoc [hunk 6]
```diff
@@ -9,8 +9,10 @@
```
Comment (petra-sargent) [inline docs/topics/modules/using_backlog_to_create_work_item.adoc]: Looks good!

File: docs/topics/modules/assigning_an_area.adoc (hunk 1 above)
Comment (mishaone) [reply docs/topics/modules/assigning_an_area.adoc]: Hi @petra-sargent. The Plan button is the one on the top of every page (except your OSIO home page). It should be the only "Plan" button no matter what you're doing in OSIO. Is this clear enough, or maybe I can add a "From the top of the page" to direct users? WDYT?

Comment (Preeticp) [reply docs/topics/modules/assigning_an_area.adoc]: Hey Misha, 
This is something I discussed earlier with Robert too. Maybe a good time to come to discuss and come to a common understanding.
For me "Analyze, Plan, and Create' seem to be tabs and not buttons. For instance when you click Plan the view it generates is more like a Tab. Buttons to me are 'Create work item' on the dashboard. Thoughts?

Comment (rkratky) [reply docs/topics/modules/assigning_an_area.adoc]: What does Todd use in his doc?

Comment (Preeticp) [reply docs/topics/modules/assigning_an_area.adoc]: Tab.

Comment (petra-sargent) [reply docs/topics/modules/assigning_an_area.adoc]: @mishaone "From the top of the page" does make it more clear.  Analyze, Plan and create do seem more like tabs to me rather than buttons.

Comment (mishaone) [reply docs/topics/modules/assigning_an_area.adoc]: I'm not sure if there's much point in saying, for example, "Click the Plan tab" since stylistically we would just say "Click Plan". The markup btn is what I'm using anytime there is an element to click (element rather than just text to click). Do we want to change this across the board and what to? "Click *Plan* to..." perhaps?

Comment (mishaone) [reply docs/topics/modules/assigning_an_area.adoc]: Clarification, since md renders asterisks as italics. In the comment above, the potential change was "Click * Plan * to..." so that the name of the tab would appear bold.

Comment (Preeticp) [reply docs/topics/modules/assigning_an_area.adoc]: "Click **Plan** to " also seems consistent with our practice of highlighting all UX text in bold. So imo this seems to be good. This also makes mentioning its position, `on top` etc redundant I suppose, since there is only one Plan tab.

Comment (rkratky) [reply docs/topics/modules/assigning_an_area.adoc]: > I'm not sure if there's much point in saying, for example, "Click the Plan tab" since stylistically we would just say "Click Plan".

I wouldn't be so sure about this. In order to make the instructions unambiguous, I would consider "Click the Plan tab". Or better, I think we can assume some common knowledge of UI principles, so we could just say: "Switch to the Plan tab". That way, we're making it clear that *tabs* are involved,  and we're telling the user how to proceed. It can be expected that users know that in order to get to some other tab, they need to click on said tab's 'tab'.

> The markup btn is what I'm using anytime there is an element to click (element rather than just text to click).

@mishaone, for the sake of clarity, I would suggest using the `btn[]` syntax *only* for real buttons. I.e. not for all clickable GUI elements, but strictly for buttons. So, for example, when describing the comment input box here on GH, there would be two buttons (Cancel, Comment) and two tabs (Write, Preview).

Comment (mishaone) [reply docs/topics/modules/assigning_an_area.adoc]: > I wouldn't be so sure about this. In order to make the instructions unambiguous, I would consider "Click the Plan tab". Or better, I think we can assume some common knowledge of UI principles, so we could just say: "Switch to the Plan tab". That way, we're making it clear that tabs are involved, and we're telling the user how to proceed. It can be expected that users know that in order to get to some other tab, they need to click on said tab's 'tab'.

The style recommendations comes from IBM (I can add the specifics tomorrow at work from my physical copy), which is generally the final say in these things. Personally, I agree with not having a clunkier sounding "Click the X thing" and would vote for a reasonable compromise of "At the top of the page, click X" and refer to the actual component as the tab (e.g. "In the Planner tab, ...") rather than referring to the UI element as a tab in the click instruction. That's my two cents, but happy to go with whatever the majority opinion is.

Comment (Preeticp) [reply docs/topics/modules/assigning_an_area.adoc]: From IBM> Label: Refer to a tab by its label, followed by tab. Example: In the Display notebook, click the Screen Saver tab.
When we mention Tab, the need to mention location ("At the top of the page") seems redundant, and also favors brevity.

Comment (mishaone) [reply docs/topics/modules/assigning_an_area.adoc]: I'm a little uncertain about calling them out as tabs at all, to be honest. Standard UI tabs should have a clear tab on top attached to the content below, with other tabs sort of grayed out or otherwise clearly not selected, which doesn't seem to be the case here (maybe it should be?).
Here are some examples off the top of a Google image search for what I would call tabs: https://www.google.com.au/search?q=UI+tab&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi1zd-W7tXXAhWErJQKHehlDdcQ_AUICigB&biw=1920&bih=1107

tl:dr: I don't dispute IBM's advice on documenting tabs, I would be uncertain about calling the top row of options in OSIO tabs at all, since they don't look the way tabs usually do in UIs and may therefore just be confusing.

Comment (mishaone) [reply docs/topics/modules/assigning_an_area.adoc]: @Preeticp and @rkratky, this has been open a while, so I'd suggest merging it and splitting off the discussion about style changes (or just a task to change this across the board) to a new issue. WDYT?

Comment (rkratky) [reply docs/topics/modules/assigning_an_area.adoc]: @mishaone, I believe this PR has already been merged.
